{
  "uid" : "4b4757e66a1912dae1a509f688f20b0f",
  "children" : [ {
    "name" : "Product defects",
    "children" : [ {
      "name" : "HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1895f3c940>: Failed to establish a new connection: [Errno 111] Connection refused')): eth_blockNumber",
      "children" : [ {
        "name" : "Latest block number test",
        "uid" : "10a571adbf5d8178",
        "parentUid" : "348e02f8efa0df2c4bd5ec371eae9e78",
        "status" : "failed",
        "time" : {
          "start" : 1641981401426,
          "stop" : 1641981402027,
          "duration" : 601
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      } ],
      "uid" : "348e02f8efa0df2c4bd5ec371eae9e78"
    }, {
      "name" : "Response status code does not indicate success: 500 (Internal Server Error).",
      "children" : [ {
        "name" : "Subsequent requests to faucet",
        "uid" : "7421f9a8d93c22d",
        "parentUid" : "947fdbd94baa05886bb5cde958459165",
        "status" : "failed",
        "time" : {
          "start" : 1641981401527,
          "stop" : 1641981402129,
          "duration" : 602
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      }, {
        "name" : "GetBalance Async",
        "uid" : "ee1849f4e0c88cbd",
        "parentUid" : "947fdbd94baa05886bb5cde958459165",
        "status" : "failed",
        "time" : {
          "start" : 1641981401708,
          "stop" : 1641981402103,
          "duration" : 395
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      }, {
        "name" : "Single request to faucet",
        "uid" : "f7e8c4da6ff486a8",
        "parentUid" : "947fdbd94baa05886bb5cde958459165",
        "status" : "failed",
        "time" : {
          "start" : 1641981402130,
          "stop" : 1641981402547,
          "duration" : 417
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      } ],
      "uid" : "947fdbd94baa05886bb5cde958459165"
    }, {
      "name" : "web3_eth_contract_1.Contract is not a constructor",
      "children" : [ {
        "name" : "simple contract deployment",
        "uid" : "e671a199c77821de",
        "parentUid" : "95946c1fb683d8dfc7058534e9f1029b",
        "status" : "failed",
        "time" : {
          "start" : 1641981408825,
          "stop" : 1641981410443,
          "duration" : 1618
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "1_Storage.sol" ]
      }, {
        "name" : "simple contract deployment",
        "uid" : "968254b11532cdd4",
        "parentUid" : "95946c1fb683d8dfc7058534e9f1029b",
        "status" : "failed",
        "time" : {
          "start" : 1641981410444,
          "stop" : 1641981411500,
          "duration" : 1056
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "3_Ballot.sol" ]
      }, {
        "name" : "simple contract deployment",
        "uid" : "6aee1bd3da2ff4c7",
        "parentUid" : "95946c1fb683d8dfc7058534e9f1029b",
        "status" : "failed",
        "time" : {
          "start" : 1641981411509,
          "stop" : 1641981411974,
          "duration" : 465
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "proof.sol" ]
      } ],
      "uid" : "95946c1fb683d8dfc7058534e9f1029b"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18955e8d60>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "the first funding request",
        "uid" : "f5b41dabb1d3d3d1",
        "parentUid" : "44ac49662b07dae03ab1c3a9e5fa4d74",
        "status" : "failed",
        "time" : {
          "start" : 1641981411975,
          "stop" : 1641981412504,
          "duration" : 529
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "1" ]
      } ],
      "uid" : "44ac49662b07dae03ab1c3a9e5fa4d74"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1895f2c820>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "the first funding request",
        "uid" : "d74ba4fc3e9cc08f",
        "parentUid" : "347a7ffca39b44e745a5e6bb29f02707",
        "status" : "failed",
        "time" : {
          "start" : 1641981412505,
          "stop" : 1641981412916,
          "duration" : 411
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "10" ]
      } ],
      "uid" : "347a7ffca39b44e745a5e6bb29f02707"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18957056d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "the second funding request",
        "uid" : "752030cbf6d19108",
        "parentUid" : "f377537e18296061c1ec9b7747a7e197",
        "status" : "failed",
        "time" : {
          "start" : 1641981412917,
          "stop" : 1641981413326,
          "duration" : 409
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "2", "3", "5" ]
      } ],
      "uid" : "f377537e18296061c1ec9b7747a7e197"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1895fdb670>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "the second funding request",
        "uid" : "908b655a0a20005c",
        "parentUid" : "8ac44fe0708dd31948b74f0e489296ca",
        "status" : "failed",
        "time" : {
          "start" : 1641981413327,
          "stop" : 1641981413678,
          "duration" : 351
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "10", "10", "20" ]
      } ],
      "uid" : "8ac44fe0708dd31948b74f0e489296ca"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189603dd90>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "a user to user successful transaction",
        "uid" : "2c9903e88fa7590",
        "parentUid" : "902ab4a93e78ef8c12b98ae12f50f8e1",
        "status" : "failed",
        "time" : {
          "start" : 1641981413678,
          "stop" : 1641981414039,
          "duration" : 361
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "1", "10" ]
      } ],
      "uid" : "902ab4a93e78ef8c12b98ae12f50f8e1"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1895fcdbb0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "a user to user successful transaction",
        "uid" : "5a98a9c21ceb5d2c",
        "parentUid" : "9eca0cd92fc5c8bb7f2f55fe0c144a95",
        "status" : "failed",
        "time" : {
          "start" : 1641981414040,
          "stop" : 1641981414499,
          "duration" : 459
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "2", "10" ]
      } ],
      "uid" : "9eca0cd92fc5c8bb7f2f55fe0c144a95"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1896138e80>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "a user to user successful transaction",
        "uid" : "387aba40587b104",
        "parentUid" : "55ebb13f2800c9b2c95bda6490d3d781",
        "status" : "failed",
        "time" : {
          "start" : 1641981414500,
          "stop" : 1641981414978,
          "duration" : 478
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "9", "10" ]
      } ],
      "uid" : "55ebb13f2800c9b2c95bda6490d3d781"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1894685760>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "a user to user failed transaction",
        "uid" : "1c3be404ad574de8",
        "parentUid" : "2eb5ea031e573b5334ac510643f63ddf",
        "status" : "failed",
        "time" : {
          "start" : 1641981414979,
          "stop" : 1641981415374,
          "duration" : 395
        },
        "flaky" : true,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "10", "10" ]
      } ],
      "uid" : "2eb5ea031e573b5334ac510643f63ddf"
    }, {
      "name" : "Returned error: HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1896028d60>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "a user to user failed transaction",
        "uid" : "c542625b6c1f0ee8",
        "parentUid" : "39fedf7025275e835edf3ba1c4ce43f9",
        "status" : "failed",
        "time" : {
          "start" : 1641981415375,
          "stop" : 1641981415744,
          "duration" : 369
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "11", "10" ]
      } ],
      "uid" : "39fedf7025275e835edf3ba1c4ce43f9"
    } ],
    "uid" : "8fb3a91ba5aaf9de24cc8a92edc82b5d"
  }, {
    "name" : "Test defects",
    "children" : [ {
      "name" : "Cannot read properties of undefined (reading 'Provider')",
      "children" : [ {
        "name" : "Connection test",
        "uid" : "acb5cfbdc18907df",
        "parentUid" : "dfabb5f02049c5380745ab97050b4aa1",
        "status" : "broken",
        "time" : {
          "start" : 1641981407326,
          "stop" : 1641981407332,
          "duration" : 6
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      } ],
      "uid" : "dfabb5f02049c5380745ab97050b4aa1"
    }, {
      "name" : "NotImplementedError: STEP: Given there is a contract '{contract_file}'\n",
      "children" : [ {
        "name" : "simple contract deployment -- @1.1 ",
        "uid" : "f62500ad10c73a5e",
        "parentUid" : "7b472810c7032364afafed009fd2c268",
        "status" : "broken",
        "time" : {
          "start" : 1641981492653,
          "stop" : 1641981492655,
          "duration" : 2
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "1_Storage.sol" ]
      }, {
        "name" : "simple contract deployment -- @1.2 ",
        "uid" : "1811f7e4c36af026",
        "parentUid" : "7b472810c7032364afafed009fd2c268",
        "status" : "broken",
        "time" : {
          "start" : 1641981492656,
          "stop" : 1641981492657,
          "duration" : 1
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "3_Ballot.sol" ]
      }, {
        "name" : "simple contract deployment -- @1.3 ",
        "uid" : "b967129006c519db",
        "parentUid" : "7b472810c7032364afafed009fd2c268",
        "status" : "broken",
        "time" : {
          "start" : 1641981492657,
          "stop" : 1641981492658,
          "duration" : 1
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "proof.sol" ]
      } ],
      "uid" : "7b472810c7032364afafed009fd2c268"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18946486d0>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "the first funding request -- @1.1 ",
        "uid" : "7495ec872af12f27",
        "parentUid" : "b0f14767e5d20221f9da32ecca519349",
        "status" : "broken",
        "time" : {
          "start" : 1641981492660,
          "stop" : 1641981493497,
          "duration" : 837
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "1" ]
      } ],
      "uid" : "b0f14767e5d20221f9da32ecca519349"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189a55c550>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "the first funding request -- @1.2 ",
        "uid" : "fb3d355325de71d4",
        "parentUid" : "6628f5663a0c47f8f5ddbc4520ca4748",
        "status" : "broken",
        "time" : {
          "start" : 1641981493498,
          "stop" : 1641981494035,
          "duration" : 537
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "10" ]
      } ],
      "uid" : "6628f5663a0c47f8f5ddbc4520ca4748"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18960539d0>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "the second funding request -- @1.1 ",
        "uid" : "7613276421fb09be",
        "parentUid" : "f30f30b54af4fca0a20fd3a8d8f014dd",
        "status" : "broken",
        "time" : {
          "start" : 1641981494036,
          "stop" : 1641981494609,
          "duration" : 573
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "2", "3", "5" ]
      } ],
      "uid" : "f30f30b54af4fca0a20fd3a8d8f014dd"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189a545250>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "the second funding request -- @1.2 ",
        "uid" : "394a50f32ae84175",
        "parentUid" : "c4fdf935d05b2888a7e75001147a99ab",
        "status" : "broken",
        "time" : {
          "start" : 1641981494609,
          "stop" : 1641981495148,
          "duration" : 539
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "10", "10", "20" ]
      } ],
      "uid" : "c4fdf935d05b2888a7e75001147a99ab"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f189a590130>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "a user to user successful transaction -- @1.1 ",
        "uid" : "7bacdf1fabef4250",
        "parentUid" : "a60871fcd76217a3fd2607b21c779373",
        "status" : "broken",
        "time" : {
          "start" : 1641981495151,
          "stop" : 1641981495665,
          "duration" : 514
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "1", "10" ]
      } ],
      "uid" : "a60871fcd76217a3fd2607b21c779373"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18961dcd00>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "a user to user successful transaction -- @1.2 ",
        "uid" : "462aa4dcd83fe322",
        "parentUid" : "e78bead57c0c3e5d60ab6d243220e605",
        "status" : "broken",
        "time" : {
          "start" : 1641981495666,
          "stop" : 1641981496180,
          "duration" : 514
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "2", "10" ]
      } ],
      "uid" : "e78bead57c0c3e5d60ab6d243220e605"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1894601850>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "a user to user successful transaction -- @1.3 ",
        "uid" : "117acc2d83db2f32",
        "parentUid" : "e12e451f72095ccead1666bd8716b136",
        "status" : "broken",
        "time" : {
          "start" : 1641981496181,
          "stop" : 1641981496749,
          "duration" : 568
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "9", "10" ]
      } ],
      "uid" : "e12e451f72095ccead1666bd8716b136"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f18960420a0>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "a user to user failed transaction -- @1.1 ",
        "uid" : "8c53ceb8b5d072bf",
        "parentUid" : "40ef5f619caf549e8fb3d4b22e91a3f3",
        "status" : "broken",
        "time" : {
          "start" : 1641981496750,
          "stop" : 1641981497261,
          "duration" : 511
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "10", "10" ]
      } ],
      "uid" : "40ef5f619caf549e8fb3d4b22e91a3f3"
    }, {
      "name" : "ValueError: {'code': -32000, 'message': \"HTTPConnectionPool(host='10.2.0.2', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1895f686d0>: Failed to establish a new connection: [Errno 111] Connection refused'))\"}\n",
      "children" : [ {
        "name" : "a user to user failed transaction -- @1.2 ",
        "uid" : "dafe79ba64f83542",
        "parentUid" : "3c23efcac5238babaf7ea0b83c8758ab",
        "status" : "broken",
        "time" : {
          "start" : 1641981497262,
          "stop" : 1641981497770,
          "duration" : 508
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ "11", "10" ]
      } ],
      "uid" : "3c23efcac5238babaf7ea0b83c8758ab"
    }, {
      "name" : "Expecting a stackmap frame at branch target 65\nException Details:\n  Location:\n    org/neonlabs/e2e/web3j/Connection.createConnection()Lorg/web3j/protocol/Web3j; @16: aload_1\n  Reason:\n    Expected stackmap frame at this location.\n  Bytecode:\n    0000000: 126d c000 6f4c b200 2e2a 2ab8 0034 3a04\n    0000010: 2b04 0454 b800 3a19 04b6 003e bb00 0759\n    0000020: b700 094d 2b05 0454 bb00 0a59 2cb6 000c\n    0000030: b700 10b8 0013 4e2b 0604 542d 3a06 a700\n    0000040: 143a 05b8 003a 1905 b600 4219 052b 0704\n    0000050: 54bf b800 3ab6 0045 1906 2b08 0454 b0  \n  Exception Handler Table:\n    bci [16, 65] => handler: 65\n",
      "children" : [ {
        "name" : "shouldConnectToNetwork()",
        "uid" : "2ea8042638812a20",
        "parentUid" : "09c597d396b21995ae0d2ffabddfffbb",
        "status" : "broken",
        "time" : {
          "start" : 1641981550778,
          "stop" : 1641981550906,
          "duration" : 128
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      }, {
        "name" : "shouldGetLatestBlockBalance()",
        "uid" : "7c3be491dbf1075b",
        "parentUid" : "09c597d396b21995ae0d2ffabddfffbb",
        "status" : "broken",
        "time" : {
          "start" : 1641981551364,
          "stop" : 1641981551517,
          "duration" : 153
        },
        "flaky" : false,
        "newFailed" : false,
        "newPassed" : false,
        "newBroken" : false,
        "retriesCount" : 0,
        "retriesStatusChange" : false,
        "parameters" : [ ]
      } ],
      "uid" : "09c597d396b21995ae0d2ffabddfffbb"
    } ],
    "uid" : "bdbf199525818fae7a8651db9eafe741"
  } ],
  "name" : "categories"
}